yarn.lock

```diff
@@ -1072,6 +1122,11 @@ acorn@^7.4.0:
   resolved "https://registry.yarnpkg.com/acorn/-/acorn-7.4.1.tgz#feaed255973d2e77555b83dbc08851a6c63520fa"
   integrity sha512-nQyp0o1/mNdbTO1PO6kHkwSrmgZ0MT/jCCpNiwbUjGoRN4dlBhqJtoQuCnEOKzgTVwg0ZWiCoQy6SxMebQVh8A==
 
+add@^2.0.6:
```

Comment:
@csm-thu so this is also to be removed?

Reply:
Yes, "yarn" and "add" dependencies must be removed from the yarn.lock file, but you shouldn't do it manually.
A proper way to achieve this would be to revert to a previous state and add only the necessary dependencies